flowmachine/tests/test_query_object_construction.py

```diff
@@ -278,13 +279,60 @@ def test_construct_query(diff_reporter):
                 "method": "last",
             },
             "to_location": {
-                "query_kind": "unique_locations",
+                "query_kind": "most_frequent_location",
```

Comment:
Why has this changed? Given that "unique_locations" is an extra option for Flows (in addition to those in ReferenceLocationSchema), i think it's useful to have "unique_locations" used in one of the tests.